templates/kubelet.service

```diff
@@ -10,7 +10,7 @@ ExecStart={{coreos_kubelet_install_dir}}/kubelet-wrapper \
   --hostname-override={{ansible_default_ipv4.address}} \
   --cluster_dns={{dns_server}} \
   --cluster_domain=cluster.local \
-  --cadvisor-port=0 \
+  --cadvisor-port=4194 \
```

Comment:
```--cadvisor-port int32      The port of the localhost cAdvisor endpoint (set to 0 to disable) (default 4194)```

To be able to disable this when needed, delete this option from `kubelet.service` completely, and add following in defaults/main.yml

```
default_kubelet_options:
  image_gc_high_threshold: 90
  image_gc_low_threshold: 80
  minimum_image_ttl_duration: "60m"
  cadvisor_port: 0
```
In base repo, we can set `cadvisor_port: 4194` in `custom_kubelet_options: {}`